### PR TITLE
Localized moneyPickedUp message

### PR DESCRIPTION
**Pull request by mercurius-00**
## What are the changes?

Localized moneyPickedUp message

## What did change?

Localized the message "You picked up ₽______!".

### Screenshots/Videos

before:
image

after:
image

## How to test the changes?

Use "Pay Day" or "Make It Rain".

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [ ] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/fr/battle.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ export const battle: SimpleTranslationEntries = {
   "switchQuestion": "Voulez-vous changer\nvotre {{pokemonName}} ?",
   "trainerDefeated": "Vous avez battu\n{{trainerName}} !",
   "moneyWon": "Vous remportez\n{{moneyAmount}} ₽ !",
+  "moneyPickedUp": "You picked up ₽{{moneyAmount}}!",
```
Comment (GoldTra) [inline src/locales/fr/battle.ts:17]:
```suggestion
  "moneyPickedUp": "Has recogido {{moneyAmount}} ₽.",
```

**Review by GoldTra — CHANGES_REQUESTED** [review]
Spanish translations

File: src/locales/ko/battle.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ export const battle: SimpleTranslationEntries = {
   "switchQuestion": "{{pokemonName}}[[를]]\n교체하시겠습니까?",
   "trainerDefeated": "{{trainerName}}[[와]]의\n승부에서 이겼다!",
   "moneyWon": "상금으로\n₽{{moneyAmount}}을 손에 넣었다!",
+  "moneyPickedUp": "You picked up ₽{{moneyAmount}}!",
```
Comment (KimJeongSun) [inline src/locales/ko/battle.ts:17]:
```suggestion
  "moneyPickedUp": "₽{{moneyAmount}}을 주웠다!",
```
image

I suggest a Korean translation as above.

File: src/locales/fr/battle.ts (hunk 1 above)
Comment (Adri1) [reply src/locales/fr/battle.ts:17]: @GoldTra You put your Spanish translation on French 😭

Comment (Adri1) [inline src/locales/fr/battle.ts:17]:
```suggestion
  "moneyPickedUp": "Vous obtenez {{moneyAmount}} ₽ !",
```

**Review by Adri1 — COMMENTED** [review]
Real French translation

File: src/locales/de/battle.ts [hunk 3]
```diff
@@ -14,6 +14,7 @@ export const battle: SimpleTranslationEntries = {
   "switchQuestion": "Möchtest du\n{{pokemonName}} auswechseln?",
   "trainerDefeated": "{{trainerName}}\nwurde besiegt!",
   "moneyWon": "Du gewinnst\n{{moneyAmount}} ₽!",
+  "moneyPickedUp": "You picked up ₽{{moneyAmount}}!",
```
Comment (CodeTappert) [inline src/locales/de/battle.ts:17]:
```suggestion
  "moneyPickedUp": "Du hebst {{moneyAmount}} ₽ auf!",
```

Source Pokémon Scarlet/Violet

**Review by sodaMelon — APPROVED** [review]

**Review by EnochG1 — APPROVED** [review]

File: src/locales/ko/battle.ts (hunk 2 above)
Comment (EnochG1) [reply src/locales/ko/battle.ts:17]: @KimJeongSun 
Minor thing but could you replace '을' to [[를]] ?

```suggestion
  "moneyPickedUp": "₽{{moneyAmount}}[[를]] 주웠다!",
```

File: src/locales/pt_BR/battle.ts [hunk 4]
```diff
@@ -14,6 +14,7 @@ export const battle: SimpleTranslationEntries = {
   "switchQuestion": "Quer trocar\nde {{pokemonName}}?",
   "trainerDefeated": "Você derrotou\n{{trainerName}}!",
   "moneyWon": "Você ganhou\n₽{{moneyAmount}} por ganhar!",
+  "moneyPickedUp": "You picked up ₽{{moneyAmount}}!",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/battle.ts:17]:
```suggestion
  "moneyPickedUp": "Você pegou ₽{{moneyAmount}} do chão!",
```

**Comment by CodeTappert** [comment]
@mercurius-00 Do you know how to add our suggestions to your PR? Or do you need help with that

**Review by EnochG1 — APPROVED** [review]

**Comment by mercurius-00** [comment]
> @mercurius-00 Do you know how to add our suggestions to your PR? Or do you need help with that

Done now. sorry for the delay

**Review by Tempo-anon — APPROVED** [review]
Approved by the wiki/tl team

**Pull request merged**